src/components/moderation/Hider.tsx

```diff
@@ -29,22 +29,22 @@ export function Outer({
   modui,
   children,
 }: React.PropsWithChildren<{
-  modui: ModerationUI
+  modui?: ModerationUI
```

Comment:
I'm loosening these types since we often are using `useModerationOpts()`, which is possibly undefined - but really shouldn't be since the context is wrapping the entire app.